### PR TITLE
Adding Mutation and Subscription 

### PR DESCRIPTION
**Pull request by pankaj-ps**
Hii Sir, Please Review my Pull Request

### CODE REVIEW

File: src/module/user/subscription.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { pubsub, UPDATE_USER } from '../../subscription';
```
Comment (pinku4444) [inline src/module/user/subscription.js]: Use pubsub from context.

File: src/module/user/mutation.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { pubsub, UPDATE_USER } from '../../subscription';
+const Mutation = {
+
+    updateUsers: async (parent, args, { dataSources }, info) => {
+        const user = await dataSources.userApi.updateUser(args.input);
+        pubsub.publish(UPDATE_USER, { UpdateUserSub: args });
```
Comment (pinku4444) [inline src/module/user/mutation.js]: please fix publish part.

**Review by pinku4444 — COMMENTED** [review]
please fix all issue.

File: src/module/user/mutation.js (hunk 2 above)
Comment (pankaj-ps) [reply src/module/user/mutation.js]: done

File: src/module/user/subscription.js (hunk 1 above)
Comment (pankaj-ps) [reply src/module/user/subscription.js]: done

**Pull request merged**